nodes/cvx/cvx.go

```diff
@@ -0,0 +1,99 @@
+package cvx
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/srl-labs/containerlab/nodes"
+	"github.com/srl-labs/containerlab/runtime"
+	"github.com/srl-labs/containerlab/types"
+	"github.com/weaveworks/ignite/pkg/operations"
+)
+
+var (
+	defaultCvxKernelImageRef  = "docker.io/networkop/kernel:4.19"
+	defaultIgniteSandboxImage = "networkop/ignite:dev"
```

Comment:
in short, what is specific about this sandbox?

Reply:
you mean why I host it in my repo? The way ignite works is they only build containers once they've shipped a release, which they haven't done yet, so I'm using a temp image hosted in my account. Once they make the next release, I'll update this to point to their image:tag.

Reply:
aha, clear. that is what I though, but wanted to double check in case anyone ask